.gitignore

```diff
@@ -102,3 +102,4 @@ dist
 
 # TernJS port file
 .tern-port
+package-lock.json
```

Comment:
package-lock should generally be part of the repo as it ensures everyone gets the same package versions when the package.json itself does not specify exact versions